sdk/appconfiguration/app-configuration/src/internal/helpers.ts

```diff
@@ -216,3 +207,35 @@ function normalizeResponse<T extends HttpResponseField<any> & { eTag?: string }>
 
   return newResponse;
 }
+
+/**
+ * Translates user-facing field names into their `select` equivalents (these can be
+ * seen in the `KnownEnum5`)
+ *
+ * @param fieldNames fieldNames from users.
+ * @returns The field names translated into the `select` field equivalents.
+ *
+ * @hidden
+ */
+export function formatFieldsForSelect(
+  fieldNames: (keyof ConfigurationSetting)[] | undefined
+): string[] | undefined {
+  if (fieldNames == null) {
```

Comment:
Nit: '==' causes eslint to be angry.

Reply:
This actually is fine in eslint. It looks like (in other cases) using `==` gets flagged because it expects `===` but I'm purposefully just lumping null and undefined into the same bucket.